javascript/javascript1/week4/js1-week4-homework/index.js

```diff
@@ -0,0 +1,92 @@
+const name = [];
+const toDo = [];
+function getReply(command) {
+  const lowerCase = command.toLowerCase();
+  //console.log(lowerCase)
+
+  let words = lowerCase.split(" ");
+  let myName = words[words.length - 1];
+  name.push(myName);
+  //console.log(name)
+  if (lowerCase.includes("hello my name is")) {
+    console.log("Nice to meet you " + name[0]);
+  }
+  if (lowerCase.includes("what is my name")) {
+    console.log("Your name is " + name[0]);
+    /* function whatIsYourName(){
+        console.log("Your name is "+name)
+       }
+         whatIsYourName(); */
+  }
+  if (lowerCase.includes("add fishing to my todo")) {
+    toDo.push("fishing");
+    console.log("fishing added to your todo");
+  }
+  if (lowerCase.includes("add singing in the shower to my todo")) {
+    toDo.push("singing in the shower");
+    console.log("singing in the shower added to your todo");
+  }
+  if (lowerCase.includes("remove fishing from my todo")) {
+    toDo.shift("fishing");
+    console.log("Removed fishing from your todo");
+  }
+  if (lowerCase.includes("what is on my todo?")) {
+    console.log(`You have ${toDo.length} todo ${toDo}`);
+  }
+  if (lowerCase.includes("what day is it today?")) {
```

Comment:
There is also the [Intl module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat).